### ISSUE TITLE
Improve readibility of unread email count

### ISSUE THREAD

**Issue by vlatkoB** [gyunaev/birdtray#32]
Hi. Let me first thank you for this tool. A life saver. 
I managed to make everything working (with mork parser, global search doesn't work), but the readibility of the unread count is quite bad. I'm using dark theme with Cinnamon on Arch, and if I have no blinking, it is (almost) impossible to read the count, as it is displayed on top of TB icon.
Maybe to have two icons, one for 0 unread emails (current), and another for X unread emails, where the latter would be dimmed, changed completely (maybe blank) or set by the user?

**Comment by gyunaev** [comment]
Have you tried to change the unread notification font color? The default color is dark blue; if you change it to white, would it improve?

**Comment by vlatkoB** [comment]
Yes, I tried several colours, but not much help. White lays over lower part of the TB icon which is, you guessed it, white, light, the upper part of the icon is blue, dark, so whichever colour I choose, half of it is not clearly readable. Red is the best, but it gets kinda lost inside the icon because the font is rather small and, even though I set the weight to 200%, still not very "bold".

**Comment by gyunaev** [comment]
Please note that you can also change the icon itself.

**Comment by vlatkoB** [comment]
You mean by changing icon is source tree? Yes, that's a possible solution, but maybe a UI would be better when you have time.

**Comment by gyunaev** [comment]
No, if you open Settings, there is "Set icon" button on the right, which opens the file dialog when you click it. Choose any other icon you'd like. No need to recompile.

**Comment by vlatkoB** [comment]
I completely missed that. :-( Thanks.
